vmdb/app/models/host_service_group.rb

```diff
@@ -0,0 +1,11 @@
+class HostServiceGroup < ActiveRecord::Base
+  has_many :filesystems, :dependent => :nullify
```

Comment:
Cna you explain the need for filesystems?  If we are grouping host services, I don't understand this part.

Additionally, if this is just a grouping of system services, then should this just be modeled as SystemServiceGroup instead of hard-modeling to Host?